pkg/keccak/Makefile.include

```diff
@@ -0,0 +1,19 @@
+INCLUDES += -I$(PKGDIRBASE)/keccak/
+
+INCLUDES += -I$(PKGDIRBASE)/keccak/Common
+INCLUDES += -I$(PKGDIRBASE)/keccak/Constructions
+INCLUDES += -I$(PKGDIRBASE)/keccak/Keyak
+INCLUDES += -I$(PKGDIRBASE)/keccak/Ketje
+INCLUDES += -I$(PKGDIRBASE)/keccak/Modes
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP/KeccakP-800-times2/OptimizedAsmARM
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP/KeccakP-800-times4/OptimizedAsmARM
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP/KeccakP-800-times8/OptimizedAsmARM
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP/KeccakP-1600-times2/OptimizedAsmARM
+INCLUDES += -I$(PKGDIRBASE)/keccak/PlSnP/KeccakP-1600-times4/OptimizedAsmARM
```

Comment:
if this PR is focusing on KeccakP-800, do we really need all these includes?
If this PR is *not* focusing on KeccakP-800 in particular, what does it aim for exactly?

Reply:
This was a remainder from the early days of experimenting, I've cleaned up this file. I will push it together with the changes to the Makefile.